mne/time_frequency/tfr.py

```diff
@@ -630,7 +631,17 @@ def _tfr_aux(method, inst, freqs, decim, return_itc, picks, average,
                                    method='%s-itc' % method))
     else:
         power = out
-        out = EpochsTFR(info, power, times, freqs, method='%s-power' % method)
+        if isinstance(inst, BaseEpochs):
+            meta = inst._metadata.copy() if inst._metadata is not None \
+                else None
+            evs = inst.events.copy()
+            ev_id = inst.event_id.copy()
```

Comment:
use

copy.deepcopy(inst.event_id)

as event_id can be just an integer or a list of integer for some epochs.